src/components/common/Typography/DPNormalize.tsx

```diff
@@ -0,0 +1,21 @@
+import { P5 } from "@deskpro/deskpro-ui";
+import styled from "styled-components";
+import { dpNormalize } from "./styles";
+import { addBlankTargetToLinks } from "../../../utils";
+import type { FC } from "react";
+
+type Props = {
+  text?: string,
+};
+
+const Text = styled(P5)`
+  width: 100%;
+
+  ${dpNormalize}
+`;
+
+const DPNormalize: FC<Props> = ({ text }) => (
+  <Text dangerouslySetInnerHTML={{ __html: addBlankTargetToLinks(text) || "-" }} />
```

Comment:
Where is the source of this `html` coming from, is it safe?

Reply:
It can be a custom property with the type `html` https://developers.hubspot.com/docs/api/crm/properties